packet-manager-default.properties

```diff
@@ -32,6 +32,12 @@ object.store.s3.url=http://minio.minio:9000
 object.store.s3.region=${s3.region}
 object.store.s3.readlimit=10000000
 
+
+##------------------------------------------------------Redis cache properties----------------------------------------------------
+redis.cache.hostname=10.20.20.90
```

Comment:
Lets add the following comment also.
## The following properties will be used only when the redis cache provider is available in the loader path of packetmanager service and spring cache is enabled for redis